tests/test_jmcomic/test_jm_plugin.py

```
@@ -0,0 +1,37 @@
+        """
+        photo_id = '350234'
+        option = self.new_option()
+
+        flawed_rule = {
+            'base_dir': option.dir_rule.base_dir,
+            'rule': '{Atitle}/{Aid}_photo.jpg'
+        }
+
+        from jmcomic.jm_downloader import DoNotDownloadImage
+
+        # 将四个需要校验的插件全部进行孤立测试，避免前一个插件后续报错导致循环终端
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)